proto/ory/keto/relation_tuples/v1alpha2/check_service.proto

```diff
@@ -35,9 +74,26 @@ message CheckRequest {
   // The relation between the Object and the Subject.
   string relation = 3 [deprecated = true];
   // The related subject in this check.
-  Subject subject = 4 [deprecated = true];
+  Subject subject = 4 [
+    deprecated = true,
+    (google.api.field_visibility).restriction = "NO_SWAGGER"
+  ];
+
+  oneof rest_api_subject {
```

Comment:
I guess this is required because the tuple in gRPC is wrapped in the tuple field. Can't we use the `body: "tuple"` annotation instead? The way I understand the docs is that we can specify which field(s) are in the body (and potentially path or query). In this case we would have the `tuple` and `max-depth` fields in the body.

Reply:
The problem is that we also have a `GET` endpoint for check, and the mapping has to handle this case as well.

With `check` we really have too many endpoints. For a next version, I would like to have only one endpoint and also define the `OpenAPI` behaviour as a flag on the request.